src/Tools/ExternalAccess/LiveShare/Microsoft.CodeAnalysis.ExternalAccess.LiveShare.csproj

```diff
@@ -11,6 +11,8 @@
       A supporting package for Live Share:
       https://devdiv.visualstudio.com/DevDiv/_git/Cascade?path=%2Fsrc%2FVS%2FLanguageService
     </PackageDescription>
+    <CreateVsixContainer>false</CreateVsixContainer>
```

Comment:
is this needed?

Reply:
yes, it seems like the <generatepkgdef> tag causes it to try and do vsix stuff (looks for source.extension.vsixmanifest) and then the build fails.  Adding that tag makes sure it doesn't look for that.

Reply:
I see. cool.